package.json

```diff
@@ -10,7 +10,8 @@
     "lint:fix": "eslint --fix --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "lint": "eslint --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "format": "npx prettier --write .",
-    "test": "ts-mocha -p tsconfig.json --recursive 'test/**/*.test*' --timeout 5000",
+    "test": "ts-mocha -p tsconfig.json --recursive 'test/**/*.test*' --exclude test/e2e/* --timeout 5000",
```

Comment:
Repurposing the existing `test` script to be our dedicated unit testing script, which from my experience, has been common. However, it is necessary to exclude the test inside the `test/e2e` directory.

Alternatively, we could re-organize these test in another fashion.